docs/3.0.0-beta.x/plugins/users-permissions.md

```diff
@@ -1,8 +1,47 @@
-# Authentication
+# Users & Permissions
 
-This Authentication API requires the Users & Permissions plugin which comes with Strapi, installed by default.
+This plugin provide a way to protect your API with a full authentication process based on JWT. This plugin comes also with an ACL strategy that allows you to manage the permissions between the groups of users.
 
-## Token usage
+To acccess to the plugin admin pannel to manage the plugin, click on the **Users & Pemissions** link in the left menu.
+
+## Concept
+
+When this plugin is installed, it adds an access layer on your application.
+The plugin use [`jwt`](https://fr.wikipedia.org/wiki/JSON_Web_Token) system to authenticate users.
+
+Each time an API request is made, we are checking if an `Authorization` header is set and strapi verify is you can acces or not to this URL.
```

Comment:
Each time an API request is sent, the server checks if an `Authorization` header is present and verifies if the user making the request has acces to the resource.